ffc-helm-library/templates/_cron-job.yaml

```diff
@@ -16,7 +16,9 @@ spec:
         metadata:
           labels:
             {{- include "ffc-helm-library.labels" . | nindent 12 }}
-            {{- if .Values.azureIdentity }}
+            {{- if and (.Values.azureIdentity) and (.Values.workloadIdentity) }}
```

Comment:
Don't think the syntax here is quite right.  Should just need one and with the two parameters listed

Reply:
You right, typo